test/Dapr.Client.Test/InvokeApiTest.cs

```diff
@@ -248,17 +263,20 @@ public async Task InvokeMethodAsync_WithNoReturnTypeAndData_UsesConfiguredJsonSe
 
             // Get Request and validate            
             httpClient.Requests.TryDequeue(out var entry).Should().BeTrue();
-            var envelope = await GrpcUtils.GetEnvelopeFromRequestMessageAsync<InvokeServiceEnvelope>(entry.Request);
+            var envelope = await GrpcUtils.GetEnvelopeFromRequestMessageAsync<InvokeServiceRequest>(entry.Request);
             envelope.Id.Should().Be("test");
-            envelope.Method.Should().Be("test");
-            var json = envelope.Data.Value.ToStringUtf8();
+            envelope.Message.Method.Should().Be("test");
 
-            json.Should().Be(JsonSerializer.Serialize(invokeRequest, jsonOptions));
+            DataWithContentType data = envelope.Message.Data.Unpack<DataWithContentType>();
+            string s = data.Body.ToStringUtf8();
```

Comment:
Please use descriptive variable names rather than s and t.
use var, be consistent with usage of var.